chart/values.yaml

```diff
@@ -0,0 +1,49 @@
+CliVersion: linkerd/cli edge-19.2.5
+ControllerComponentLabel: linkerd.io/control-plane-component
+ControllerImage: gcr.io/linkerd-io/controller:edge-19.2.5
+ControllerLogLevel: info
+ControllerReplicas: 1
+ControllerUID: 2103
+CreatedByAnnotation: linkerd.io/created-by
```

Comment:
@alpeb @ihcsim what happens when an end user changes this? Would it break their install? I imagine there's some other possibilities in here that could break things ...

Reply:
Yeah, it seems it's not necessary (nor safe) to expose all these vars. Possible alternatives that come to mind:
- Put all those that are "internal" under a separate section, trusting the user won't touch them
- Move our current charts out of the `chart` directory, and populate the `chart` directory during build time with charts with their internal values replaced.

Maybe there's some Helm trickery I'm not aware of, that can help here as well...

Reply:
For now, I think it is okay. We should just add comments to this file and group the ones that are internal. The end goal would be to have a build time creation with the correct values.

Reply:
I created an `Internal` section of `values.yaml` and added a comment. Which other values do we think belong in there?